RecoTracker/TkSeedGenerator/plugins/SeedProducerFromSoA.cc

```diff
@@ -167,4 +173,11 @@ void SeedProducerFromSoA::produce(edm::StreamID streamID, edm::Event& iEvent, co
   iEvent.put(std::move(result));
 }
 
+using SeedProducerFromSoA = SeedProducerFromSoAT<pixelTopology::Phase1>;
 DEFINE_FWK_MODULE(SeedProducerFromSoA);
+
+using SeedProducerFromSoAPhase1 = SeedProducerFromSoAT<pixelTopology::Phase1>;
+DEFINE_FWK_MODULE(SeedProducerFromSoAPhase1);
```

Comment:
This is duplicating the one above... do you plan to have them both in order to switch all configuration to the new one first, and _then_ remove the old one ?

Reply:
Yes, the idea was to keep them both in order to allow HLT to do the migration smoothly. But actually in this specific case the old could be directly removed since I don't see it used at HLT. I would anyway cleanup all the duplicates for all the modules in a followup PR.